### PR TITLE
Use as primitive

### PR DESCRIPTION
**Pull request by gossi**
PR for #18 

This is was what I would call "heuristic programming" 😂 Trying to understand AST nodes from glimmer-vm repo and applying it here.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export default type Element = HTMLElement;
```
Comment (gossi) [inline index.d.ts]: I'm pretty sure this is not correct. At least I wanted to have a specifc type imported from `ember-element-helper` to be used for TS interfaces as shown in the readme

File: lib/element-helper-syntax-plugin.js [hunk 2]
```diff
@@ -31,37 +31,18 @@ module.exports = class ElementHelperSyntaxPlugin {
 
       MustacheStatement(node) {
         if (isElementHelper(node, locals)) {
-          throw error('cannot be appended to the DOM directly', node);
+          const { path, params, hash } = transformParts(node, b);
+
+          return b.mustache(path, params, hash, false, node.loc);
```
Comment (gossi) [inline lib/element-helper-syntax-plugin.js]: Intead of returning `transformParts()` as object, they could also be returned as array and then:

```ts
return b.mustache(...[...transformParts(node, b), false, node.loc]);
```

yet this didn't provide the best readability to me.

**Comment by chancancode** [comment]
Merged in #22

**Comment by chancancode** [comment]
Thanks @gossi! I'll release soon

**Pull request closed without merging**